lib/cryptoexchange/exchanges/bigone/services/pairs.rb

```diff
@@ -0,0 +1,23 @@
+module Cryptoexchange::Exchanges
+  module Bigone
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Bigone::Market::API_URL}"
+
+        def fetch
+          output = super
+          market_pairs = []
+          output.each do |pair|
+            market_pairs << Cryptoexchange::Models::MarketPair.new(
```

Comment:
looking at the API response, I think you should be able to access the base/quote using the following

```
output['data'].each do |pair|
  pair['base']
  pair['quote']
end
```